internal/service/project/data_source_projects.go

```diff
@@ -163,6 +163,11 @@ func (d *ProjectsDS) Schema(ctx context.Context, req datasource.SchemaRequest, r
 								},
 							},
 						},
+						"tags": schema.MapAttribute{
+							ElementType: types.StringType,
+							Optional:    true,
+							Computed:    true,
```

Comment:
I guess we are removing this after yesterday's conversation? @maastha @EspenAlbert

Reply:
This is data source so this would be Computed-only